src/Components/Artwork/Details.tsx

```diff
@@ -140,7 +140,7 @@ const HighDemandInfo = () => {
     <Flex flexDirection="row" alignItems="center">
       <HighDemandIcon width={16} height={16} />
       <Text variant="xs" color="blue100" ml={0.3}>
-        High Demand
+        &nbsp;High Demand
```

Comment:
**Suggestion:** should we use `ml` instead? 👀

Reply:
I've had this discussion before and someone suggested using the `&nbsp;` instead because it's not really meant to be a spacing between components.